docs/source/addons/i18n.md

```diff
@@ -1,10 +1,10 @@
 ---
 myst:
   html_meta:
-    "description": "Internationalize your add-on and override translations"
-    "property=og:description": "Internationalize your add-on and override translations"
-    "property=og:title": "Add-on Internationalization"
-    "keywords": "Internationalization, i18n, add-on"
+    'description': 'Internationalize your add-on and override translations'
+    'property=og:description': 'Internationalize your add-on and override translations'
+    'property=og:title': 'Add-on Internationalization'
+    'keywords': 'Internationalization, i18n, add-on'
 ---
```

Comment:
Please revert the above changes, and adjust your editor's settings not to reformat quotes.